modules/aws_account_permissions/variables.tf

```
@@ -0,0 +1,32 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Required PARAMETERS
+# These variables have default values and can be modified based on the requirement.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+variable "iam_role_name" {
+  description = "Customize the name of IAM role for Illumio AWS integration"
+  type        = string
+  default     = "IllumioCloudIntegrationRole"
+  validation {
+    condition     = length(var.iam_role_name) > 1
+    error_message = "The IAM role name cannot be empty."
```

Be consistent with the Terraform docs' style.
```suggestion
    error_message = "The iam_role_name value must not be empty."
```

---

I fixed this.